CHANGELOG.md

```diff
@@ -14,7 +14,9 @@ _None_
 
 ### New Features
 
-_None_
+* Accept `LosslessStringConvertible` input for strings filters.  
+  [Antondomashnev](https://github.com/antondomashnev)
+  [#59](https://github.com/SwiftGen/StencilSwiftKit/pull/63)
```

Comment:
Please correct this.

Reply:
Thanks, I've also corrected previous entry, I think we missed it during the review of #63.
Check ad8c792

Reply:
Thanks, I've also corrected previous entry, I think we missed it during the review of #63.
Check ad8c792